library/fixbook/FixBookControl.java

```diff
@@ -12,7 +12,7 @@
 	private Book currentBook;
 
 
-	public fixbookControl() {
+	public FixBookControl() {
```

Comment:
it should be fixBookControl(). The first letter should not be capital

Reply:
@BLakshmiPrasanna Rework needed for particular error raised by reviewer Haris Ali

Reply:
@abhimaniac4 Fixed the issue please review